src/common/eeprom.cpp

```diff
@@ -103,6 +105,8 @@ static const eeprom_entry_t eeprom_map[] = {
     { "LAN_HOSTNAME",    VARIANT8_PCHAR, LAN_HOSTNAME_MAX_LEN + 1, 0 }, // EEVAR_LAN_HOSTNAME
     { "TIMEZONE",        VARIANT8_I8,    1, 0 }, // EEVAR_TIMEZONE
     { "SOUND_MODE",      VARIANT8_UI8,   1, 0 }, // EEVAR_SOUND_MODE
+    { "SOUND_VOLUME",      VARIANT8_UI8,   1, 0 }, // EEVAR_SOUND_VOLUME
+    { "LANGUAGE",					VARIANT8_UI16,   1, 0 }, // EEVAR_LANGUAGE
```

Comment:
why is this code format so weird?

Reply:
Clang format if in this file and I did not know about that - sorry